spec/integration/cds_importer/entity_mapper_spec.rb

```diff
@@ -90,5 +92,72 @@
       mapper.import
       expect(AdditionalCode.last.filename).to eq 'test.gzip'
     end
+
+    context 'when measureExcludedGeographicalArea changes are present' do
```

Comment:
It think we're missing one spec for deletion

```ruby
context 'when there are existing exclusions for this measure' do
  it 'does hard delete the exclusions' do
  end
end
```

Reply:
yep